sktime/forecasting/compose/_ignore_x.py

```diff
@@ -48,21 +45,8 @@ def __init__(self, forecaster, ignore_x=True):
 
         self.forecaster_ = forecaster.clone()
 
-        TAGS_TO_CLONE = [
-            "capability:insample",
-            "capability:pred_int",
-            "capability:pred_int:insample",
-            "handles-missing-data",
-            "requires-fh-in-fit",
-            "X-y-must-have-same-index",
-            "enforce_index_type",
-        ]
-
-        self.clone_tags(forecaster, TAGS_TO_CLONE)
-
-        # this ensures that we convert in the inner estimator, not in the multiplexer
-        self.set_tags(**{"y_inner_mtype": ALL_TIME_SERIES_MTYPES})
-        self.set_tags(**{"X_inner_mtype": ALL_TIME_SERIES_MTYPES})
+        self._set_delegated_tags(self.forecaster_)
+        self.set_tags(**{"ignores-exogeneous-X": True})
```

Comment:
I don't think you need this line since you already set this in the class' `_tags` dictionary

Reply:
I do, because `_set_delegated_tags` overwrites it again! The idea is, it copies all the behavioural ones, so gives a clean slate for delegation. In deviation to it, we need to force `ignorse-exogeneous-X` to `False`.